component/otelcol/exporter/prometheus/internal/convert/convert.go

```diff
@@ -344,18 +344,15 @@ func (conv *Converter) getOrCreateSeries(res *memorySeries, scope *memorySeries,
 		model.MetricNameLabel, name,
 		model.JobLabel, res.metadata[model.JobLabel],
 		model.InstanceLabel, res.metadata[model.InstanceLabel],
+		scopeNameLabel, scope.metadata[scopeNameLabel],
+		scopeVersionLabel, scope.metadata[scopeVersionLabel],
```

Comment:
We should have a new config option to add these labels rather than always adding them.